File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/MdiFormMenuItemInfo.java
	package nta.med.data.model.ihis.system;

public class MdiFormMenuItemInfo {
	private Double menuLevel;
    private String menuTp;
    private String pgmNm;
    private String trId;
    private String pgmId;
    private String pgmSysId;
    private Double pgmEntGrad;
    private Double pgmUpdGrad;
    private String pgmScrt;
    private String pgmDupYn;
    private String pgmOpenTp;
    private String shortCut;
    private String asmName;
    private String asmPath;
    private String asmVer;
    private String menuParam;
	public MdiFormMenuItemInfo(Double menuLevel, String menuTp, String pgmNm,
			String trId, String pgmId, String pgmSysId, Double pgmEntGrad,
			Double pgmUpdGrad, String pgmScrt, String pgmDupYn,
			String pgmOpenTp, String shortCut, String asmName, String asmPath,
			String asmVer, String menuParam) {
		super();
		this.menuLevel = menuLevel;
		this.menuTp = menuTp;
		this.pgmNm = pgmNm;
		this.trId = trId;
		this.pgmId = pgmId;
		this.pgmSysId = pgmSysId;
		this.pgmEntGrad = pgmEntGrad;
		this.pgmUpdGrad = pgmUpdGrad;
		this.pgmScrt = pgmScrt;
		this.pgmDupYn = pgmDupYn;
		this.pgmOpenTp = pgmOpenTp;
		this.shortCut = shortCut;
		this.asmName = asmName;
		this.asmPath = asmPath;
		this.asmVer = asmVer;
		this.menuParam = menuParam;
	}
	public Double getMenuLevel() {
		return menuLevel;
	}
	public void setMenuLevel(Double menuLevel) {
		this.menuLevel = menuLevel;
	}
	public String getMenuTp() {
		return menuTp;
	}
	public void setMenuTp(String menuTp) {
		this.menuTp = menuTp;
	}
	public String getPgmNm() {
		return pgmNm;
	}
	public void setPgmNm(String pgmNm) {
		this.pgmNm = pgmNm;
	}
	public String getTrId() {
		return trId;
	}
	public void setTrId(String trId) {
		this.trId = trId;
	}
	public String getPgmId() {
		return pgmId;
	}
	public void setPgmId(String pgmId) {
		this.pgmId = pgmId;
	}
	public String getPgmSysId() {
		return pgmSysId;
	}
	public void setPgmSysId(String pgmSysId) {
		this.pgmSysId = pgmSysId;
	}
	public Double getPgmEntGrad() {
		return pgmEntGrad;
	}
	public void setPgmEntGrad(Double pgmEntGrad) {
		this.pgmEntGrad = pgmEntGrad;
	}
	public Double getPgmUpdGrad() {
		return pgmUpdGrad;
	}
	public void setPgmUpdGrad(Double pgmUpdGrad) {
		this.pgmUpdGrad = pgmUpdGrad;
	}
	public String getPgmScrt() {
		return pgmScrt;
	}
	public void setPgmScrt(String pgmScrt) {
		this.pgmScrt = pgmScrt;
	}
	public String getPgmDupYn() {
		return pgmDupYn;
	}
	public void setPgmDupYn(String pgmDupYn) {
		this.pgmDupYn = pgmDupYn;
	}
	public String getPgmOpenTp() {
		return pgmOpenTp;
	}
	public void setPgmOpenTp(String pgmOpenTp) {
		this.pgmOpenTp = pgmOpenTp;
	}
	public String getShortCut() {
		return shortCut;
	}
	public void setShortCut(String shortCut) {
		this.shortCut = shortCut;
	}
	public String getAsmName() {
		return asmName;
	}
	public void setAsmName(String asmName) {
		this.asmName = asmName;
	}
	public String getAsmPath() {
		return asmPath;
	}
	public void setAsmPath(String asmPath) {
		this.asmPath = asmPath;
	}
	public String getAsmVer() {
		return asmVer;
	}
	public void setAsmVer(String asmVer) {
		this.asmVer = asmVer;
	}
	public String getMenuParam() {
		return menuParam;
	}
	public void setMenuParam(String menuParam) {
		this.menuParam = menuParam;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/MdiFormMenuItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
	public class MdiFormMenuItemInfo
	{
		private String _menuLevel;
		private String _menuTp;
		private String _pgmNm;
		private String _trId;
		private String _pgmId;
		private String _pgmSysId;
		private String _pgmEntGrad;
		private String _pgmUpdGrad;
		private String _pgmScrt;
		private String _pgmDupYn;
		private String _pgmOpenTp;
		private String _shortCut;
		private String _asmName;
		private String _asmPath;
		private String _asmVer;
		private String _menuParam;

		public String MenuLevel
		{
			get { return this._menuLevel; }
			set { this._menuLevel = value; }
		}

		public String MenuTp
		{
			get { return this._menuTp; }
			set { this._menuTp = value; }
		}

		public String PgmNm
		{
			get { return this._pgmNm; }
			set { this._pgmNm = value; }
		}

		public String TrId
		{
			get { return this._trId; }
			set { this._trId = value; }
		}

		public String PgmId
		{
			get { return this._pgmId; }
			set { this._pgmId = value; }
		}

		public String PgmSysId
		{
			get { return this._pgmSysId; }
			set { this._pgmSysId = value; }
		}

		public String PgmEntGrad
		{
			get { return this._pgmEntGrad; }
			set { this._pgmEntGrad = value; }
		}

		public String PgmUpdGrad
		{
			get { return this._pgmUpdGrad; }
			set { this._pgmUpdGrad = value; }
		}

		public String PgmScrt
		{
			get { return this._pgmScrt; }
			set { this._pgmScrt = value; }
		}

		public String PgmDupYn
		{
			get { return this._pgmDupYn; }
			set { this._pgmDupYn = value; }
		}

		public String PgmOpenTp
		{
			get { return this._pgmOpenTp; }
			set { this._pgmOpenTp = value; }
		}

		public String ShortCut
		{
			get { return this._shortCut; }
			set { this._shortCut = value; }
		}

		public String AsmName
		{
			get { return this._asmName; }
			set { this._asmName = value; }
		}

		public String AsmPath
		{
			get { return this._asmPath; }
			set { this._asmPath = value; }
		}

		public String AsmVer
		{
			get { return this._asmVer; }
			set { this._asmVer = value; }
		}

		public String MenuParam
		{
			get { return this._menuParam; }
			set { this._menuParam = value; }
		}

		public MdiFormMenuItemInfo() { }

		public MdiFormMenuItemInfo(String menuLevel, String menuTp, String pgmNm, String trId, String pgmId, String pgmSysId, String pgmEntGrad, String pgmUpdGrad, String pgmScrt, String pgmDupYn, String pgmOpenTp, String shortCut, String asmName, String asmPath, String asmVer, String menuParam)
		{
			this._menuLevel = menuLevel;
			this._menuTp = menuTp;
			this._pgmNm = pgmNm;
			this._trId = trId;
			this._pgmId = pgmId;
			this._pgmSysId = pgmSysId;
			this._pgmEntGrad = pgmEntGrad;
			this._pgmUpdGrad = pgmUpdGrad;
			this._pgmScrt = pgmScrt;
			this._pgmDupYn = pgmDupYn;
			this._pgmOpenTp = pgmOpenTp;
			this._shortCut = shortCut;
			this._asmName = asmName;
			this._asmPath = asmPath;
			this._asmVer = asmVer;
			this._menuParam = menuParam;
		}

	}
}